### ISSUE TITLE
Storybook won't work with Create React App on a fresh install Error: Cannot find module 'babel-loader'

### ISSUE THREAD

**Issue by skhaz** [storybookjs/storybook#13597]
**Describe the bug**
I am trying to create a new Storybook based on create react app and fails every time I have tried.

**To Reproduce**
``` bash
$ npx create-react-app storybook-test
$ cd storybook-test
$ npx sb init
$ yarn storybook
yarn run v1.22.5
$ start-storybook -p 6006 -s public
info @storybook/react v6.1.11
info 
info => Loading static files from /home/ubuntu/workspace/foobar/storybook-test/public and serving at /.
info => Using prebuilt manager
info => Loading presets
info => Loading 1 config file in "./.storybook"
info => Loading 7 other files in "./.storybook"
info => Adding stories defined in ".storybook/main.js"
ERR! Error: Cannot find module 'babel-loader'
ERR! Require stack:
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/addon-docs/dist/frameworks/common/preset.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/presets.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/config.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-static.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/server.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/react/dist/server/index.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/react/bin/index.js
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
ERR!     at Function.resolve (internal/modules/cjs/helpers.js:80:19)
ERR!     at Object.webpack (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/addon-docs/dist/frameworks/common/preset.js:143:27)
ERR!     at /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/presets.js:259:28
ERR!     at async startPreview (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/dev-server.js:407:23)
ERR!     at async Promise.all (index 0)
ERR!     at async storybookDevServer (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/dev-server.js:494:29)
ERR!     at async buildDevStandalone (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-dev.js:317:33)
ERR!     at async buildDev (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-dev.js:380:3)
ERR!  Error: Cannot find module 'babel-loader'
ERR! Require stack:
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/addon-docs/dist/frameworks/common/preset.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/presets.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/config.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-static.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/server.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/react/dist/server/index.js
ERR! - /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/react/bin/index.js
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
ERR!     at Function.resolve (internal/modules/cjs/helpers.js:80:19)
ERR!     at Object.webpack (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/addon-docs/dist/frameworks/common/preset.js:143:27)
ERR!     at /home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/presets.js:259:28
ERR!     at async startPreview (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/dev-server.js:407:23)
ERR!     at async Promise.all (index 0)
ERR!     at async storybookDevServer (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/dev-server.js:494:29)
ERR!     at async buildDevStandalone (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-dev.js:317:33)
ERR!     at async buildDev (/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-dev.js:380:3) {
ERR!   code: 'MODULE_NOT_FOUND',
ERR!   requireStack: [
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/addon-docs/dist/frameworks/common/preset.js',
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/presets.js',
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/config.js',
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/dist/server/build-static.js',
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/core/server.js',
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/react/dist/server/index.js',
ERR!     '/home/ubuntu/workspace/foobar/storybook-test/node_modules/@storybook/react/bin/index.js'
ERR!   ]
ERR! }

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.
```

**Expected behavior**
Run `yarn storybook` without any error.

**System**
``` bash
Environment Info:

  System:
    OS: Linux 4.19 Ubuntu 20.04.1 LTS (Focal Fossa)
    CPU: (20) x64 Intel(R) Core(TM) i9-10900K CPU @ 3.70GHz
  Binaries:
    Node: 12.20.0 - ~/.nvm/versions/node/v12.20.0/bin/node
    Yarn: 1.22.5 - /usr/bin/yarn
    npm: 6.14.8 - ~/.nvm/versions/node/v12.20.0/bin/npm
```

**Additional context**
Using WSL2

**Comment by michaelryancaputo** [comment]
This is not the solution, but I was able to temporarily fix this by including `"babel-loader": "8.1.0"` in my `devDependencies`.

**Comment by michaelryancaputo** [comment]
Looks like updating to storybook `6.1.14` may have fixed this issue.
